src/styles/blocks/header.scss

```diff
@@ -38,5 +38,10 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition-duration: 300ms;
+
+    &:hover {
+      color: #00acdc;
+    }
```

Comment:
here the same
<img width="1132" alt="Screenshot 2022-12-14 at 20 35 00" src="https://user-images.githubusercontent.com/34283676/207681502-59f20450-a5a6-4373-8626-7a73aa83c192.png">